docs/application/native/guides/location-sensors/maps.md

```diff
@@ -1,5 +1,7 @@
 # Maps and Maps Service
 
+(This module has been deprecated since Tizen 8.0)
+
```

Comment:
```suggestion
[!NOTE]
All maps-service APIs have been deprecated since Tizen 8.0 and will be removed after two releases without any alternatives.
```